modules/ztp-using-pgt-to-configure-high-performance-mode.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-using-pgt-to-configure-high-performance-mode_{context}"]
+= Configuring high-performance mode using PolicyGenTemplate CRs
+
+Follow this example to set high performance mode by updating the `workloadHints` fields in the generated `PerformanceProfile` CR for the reference configuration, based on the `PolicyGenTemplate` CR in the `group-du-sno-ranGen.yaml`.
+
+High performance mode provides ultra low latency at the highest power consumption.
+
+.Prerequisites
+
+* You have configured the BIOS with performance related settings by following the guidance in "Configuring host firmware for low latency and high performance".
```

Comment:
Is it possible to add a hyperlink to the docs page referred to here?

Reply:
Can't add cross ref hyperlink unfortunately  so I have to use the additional resources section